package.json

```diff
@@ -90,13 +90,15 @@
     "@sharegate/stylelint-config-recommended": "2.1.1",
     "@storybook/addon-a11y": "6.5.13-alpha.0",
     "@storybook/addon-essentials": "6.5.13-alpha.0",
+    "@storybook/addon-interactions": "^6.5.13",
     "@storybook/react": "6.5.13-alpha.0 ",
+    "@storybook/testing-library": "^0.0.13",
```

Comment:
Is it required?

I tryed this SB feature lately and I doubt this is something we want to invest into at the moment.

Reply:
Do you actually the addon-interactions for this?

My guess is that is can be done only with a play function and storybook testing library implementation.

In fact, I wonder if you really even need to use storybook testing library implementation to do it.

Reply:
> Do you actually the addon-interactions for this?
> 
> My guess is that is can be done only with a play function and storybook testing library implementation.
> 
> In fact, I wonder if you really even need to use storybook testing library implementation to do it.

You are right, the addon-interaction is of no use.